app/jobs/super_good/solidus_taxjar/replace_transaction_job.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module SuperGood
+  module SolidusTaxjar
+    class ReplaceTransactionJob < ApplicationJob
```

Comment:
Yes, it might be nice of us to mention in documentation that a background job library like Sidekiq should be used with this extension.

I will create a new issue to track this.

Reply:
Oh looks like you did this ✨ 

#159